### ISSUE TITLE
Reader: Facebook video links vanish

### ISSUE THREAD

**Issue by designsimply** [Automattic/wp-calypso#934]
Steps to reproduce:

1. Publish a new post containing a Facebook video link on its own line
1. View the post in the Reader

<strong>Result:</strong> If a Facebook video URL is included in a post, we currently strip it out completely.

Example content:

```
https://www.facebook.com/Engineering/videos/10153414335177200/?fref=nf

Looks impressive
```

Notes from @bluefuton 

> The URL is converted to a fb-video element, but this isn't currently displayed in the Reader, so the user can't see the video or the link URL.
> 
> <img width="682" alt="screen shot 2015-07-31 at 10 18 22" src="https://cloud.githubusercontent.com/assets/17325/9012933/93d4109c-376d-11e5-9efe-5bf8b6092ddd.png">
> 
> <img width="734" alt="screen shot 2015-07-31 at 10 12 44" src="https://cloud.githubusercontent.com/assets/17325/9012934/966b3876-376d-11e5-8e83-14d328025738.png">
> 
> We could either include the FB Javascript SDK in Reader so that the fb-video divs are parsed (as per https://developers.facebook.com/docs/plugins/embedded-video-player), or we could make API changes so that Facebook video URLs aren't automagically converted to fb-video elements.

Notes from @blowery

> It's coming back from the API that way
> 
> ![screenshot](https://cldup.com/gS-cjTOqIl.png)
> 
> Peeked into wp-content/mu-plugins/shortcodes/facebook.php and it appears we're assuming the client has the facebook fbxml parser loaded.
> 
> https://developers.facebook.com/docs/plugins/embedded-video-player#wordpress
> https://developers.facebook.com/docs/plugins/embedded-video-player#

**Comment by blowery** [comment]
Would be fixed by #935

**Comment by blowery** [comment]
Closing this as we'll track it as part of #935
